ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -224,6 +231,15 @@ def get_dashboard_details(self, dashboard: Any) -> Any:
         Get Dashboard Details
         """
 
+    def yield_datamodel(self, _) -> Any:
```

Comment:
I think instead of `Any` we shall put `Optional[Iterable[CreateDashboardDataModelRequest]]` as return type